lib/rollup/jobs/trends_job.ex

```diff
@@ -0,0 +1,63 @@
+defmodule Castle.Rollup.Jobs.Trends do
+  # @redis Application.get_env(:castle, :redis)
+  import Castle.Redis.PartitionCache
+
+  # old - all days before today
+  # new - just data for today
+  def run() do
+    partition "rollup.trends", &Castle.Rollup.Jobs.Trends.combine/1, [
+      {ttl_eod(Timex.now()), &Castle.Rollup.Jobs.Trends.old_part/0},
```

Comment:
So at the end of every day, this `old_data` key expires, and it re-queries for the past 14 days of downloads.

Otherwise, it just queries for "today".